File: count2billion.cpp
#include <iostream>
#include <chrono>  // For timing

int main() {
    // Start the timer using chrono
    auto start = std::chrono::high_resolution_clock::now();

    // Loop to count to one billion
    unsigned long long count = 0;
    for (unsigned long long i = 0; i < 1000000000; ++i) {
        count++;
    }

    // Stop the timer
    auto end = std::chrono::high_resolution_clock::now();

    // Calculate the duration
    std::chrono::duration<double> elapsed = end - start;

    // Output the elapsed time in seconds
    std::cout << "Time taken to count to one billion: " << elapsed.count() << " seconds" << std::endl;

    return 0;
}

File: count2billion.py
import time

def count_to_one_billion():
    start_time = time.time()  # Start the timer
    count = 0
    for i in range(1, 1_000_000_001):
        count += 1
    end_time = time.time()  # Stop the timer
    elapsed_time = end_time - start_time
    print(f"Time taken to count to one billion: {elapsed_time:.2f} seconds")

# Run the program
count_to_one_billion()

